generator/src/test/java/com/scottlogic/deg/generator/cucumber/BeforeOrAt.feature

```diff
@@ -68,40 +68,42 @@ Scenario: Running beforeOrAt request that includes temporal field with date and
        | 2012-09-01T12:01:02.000 |
        | 2024-07-01T12:01:03.000 |
 
-@ignore #issue related to 611
 Scenario: Running beforeOrAt request against a non-contradicting beforeOrAt constraint should be successful
      Given foo is before or at 2019-01-01T00:00:00.000
        And foo is before or at 2018-01-01T00:00:00.000
        And the generator can generate at most 5 rows
      Then the following data should be generated:
        | foo                     |
-       | 2017-10-01T00:00:00.000 |
-       | 2016-10-02T00:00:00.000 |
-       | 2018-01-01T00:00:00.000 |
-       | 2019-01-01T00:00:00.000 |
+       | 0001-01-01T00:00:00.000 |
+       | 0002-01-01T00:00:00.000 |
+       | 0003-01-01T00:00:00.000 |
+       | 0004-01-01T00:00:00.000 |
+       | 0005-01-01T00:00:00.000 |
```

Comment:
start at 2018-01-01T00:00:00.000 and work down dates at a granularity of milisecond.